handlers/services.go

```diff
@@ -0,0 +1,30 @@
+package handlers
+
+import (
+	"fmt"
+	"net/http"
+
+	"github.com/gorilla/mux"
+	"github.com/swift-sunshine/swscore/log"
+	"github.com/swift-sunshine/swscore/models"
```

Comment:
Question, what is the purpose of the models package ?
The models used for the REST endpoints or something more general.
Just to sync, I'm going to send a PR and models I used are under pkg package.
I guess these are two different things and better to comment on the code instead of theorical, but just if I need to fix something before to send my PR.

Reply:
I was thinking about models as where all the business logic relay on. Where the manipulation of the data from data streams are made in order to be available from handlers.
Also, I think that models have to be the entity which is called from controllers in order to retrieve the information from backends.
I would say that go ahead with the pull request and we open discussion there about how the integration would be. In addition, I would like to wait to team feedback here. Maybe somebody brings up an architectural change.